kratos/includes/properties.h

```diff
@@ -389,6 +397,7 @@ class Properties : public IndexedObject
 
     ContainerType mData;
     TablesContainerType mTables;
+    Properties::Pointer mpAccessor;
```

Comment:
why a "Properties::Pointer"? i would expect the accessor to be of a different type

Reply:
I will do this in the next iteration